src/components/steps/StepOnboardingFormData.tsx

```diff
@@ -362,6 +365,37 @@ export default function StepOnboardingFormData({
     },
   });
 
+  const verifyVatNumber = async () => {
+    setIsVatRegistrated(false);
```

Comment:
Why did you set this to false?

Reply:
To make sure to reset the value to the inital state before the call of the api. This way i dont have old value of setState.